### ISSUE TITLE
Pagination page 1 problem

### ISSUE THREAD

**Issue by montabou** [symfony2admingenerator/GeneratorBundle#276]
On list view if I go to page 2 everything is OK but if I try to go back to page 1 it stays on page 2.
The link is ```?page=2``` for page 2 but the parameter is empty for page 1.
I don't know if the problem is here
https://github.com/symfony2admingenerator/GeneratorBundle/blob/fda9f4bc9d6ac4922cef75d64c184237b5ad0283/Resources/templates/CommonAdmin/ListAction/pager.php.twig#L8
```php
if ($this->request->query->get('page')) {
    $this->setPage($this->request->query->get('page'));
} else {
    $this->setPage(1);
}
```

**Comment by bobvandevijver** [comment]
This is already fixed in 33694db (see #266). Problem was with the PagerfantaBundle, you will need to have >=1.0.6.

Note that this commit is not yet in a release (I'm waiting on #265 before releasing 2.1). However, simply updating the PagerfantaBundle yourself should do the trick.

**Comment by montabou** [comment]
Oups sorry, using PagerfantaBundle 1.0.6 works. Thx
